### ISSUE TITLE
CLI is slow

### ISSUE THREAD

**Issue by iron3oxide** [HeartPredict/HeartPredict#52]
Even the `version` command takes about 2 seconds. I suspect this stems from us not using cached backend and data objects.

**Comment by HokageM** [comment]
I can live with that

**Comment by HokageM** [comment]
But the version command does not create objects.

**Comment by iron3oxide** [comment]
Exactly, typer has to collect all the commands and options first and I suspect that the slowdown happens there

**Comment by HokageM** [comment]
Ahh okay, but then I did not see the advantages of typer over argparse

**Comment by iron3oxide** [comment]
The advantages are convenience and elegance in how we build the CLI, not necessarily performance I think.
